user/Lambda.py

```diff
@@ -67,12 +55,30 @@ def Lambda_Acquire(acquire_time=0.1, acquire_period=0.11,num_images=100,file_nam
     @bpp.stage_decorator([scaler1])
     @bpp.monitor_during_decorator(monitored_things)
     def inner():
+        # write metadata to the dm_pars (various registers)
+        yield from bps.mv(
+            dm_pars.root_folder, file_path,
+            dm_pars.parent_folder, os.path.dirname(file_path),
+            dm_pars.data_folder, file_name,
+            dm_pars.datafilename, areadet.get_plugin_file_name(),
+            # TODO: what else?
+        )
+
         md = {
             "file_name": file_name,
             "file_path": file_path
         }
-        yield from bps.mv(scaler1.count,"Count")
-        yield from bp.count([adlambda], md=md)
+        yield from bps.mv(scaler1.count, "Count")
+        yield from bp.count([areadet], md=md)
+
```

Comment:
set the post acq metadata

Reply:
examine TODO items in `update_metadata_postscan()`

Reply:
done, needs discussion